view-samples/default-hover-cards/README.md

```diff
@@ -0,0 +1,38 @@
+# Defalut hover card on Lists
+
+## Summary
+The following image shows a list with a defalut hover added to a formatted view in a list
+
+On hover on a formatted view a file card is displayed 
+
+![screenshot of the sample](./File_hover.png)
+
+## View requirements
+- This format can be applied to any view type
```

Comment:
The sample is restricted to DocLibs.

Also, the sample contains both hover cards - profile card and file hover card

Reply:
Updated